nevergrad/optimization/base.py

```diff
@@ -116,17 +59,15 @@ class CandidateMaker:
     Note
     ----
     An instance of this class is linked to each optimizer (optimizer.create_candidate).
-    Candidates can then easily be created through: optimizer.create_candidate.from_data(data)
+    p.Instrumentations can then easily be created through: optimizer.create_candidate.from_data(data)
     and/or optimizer.create_candidate.from_call(*args, **kwargs).
     """
 
-    def __init__(self, instrumentation: parameter.Instrumentation) -> None:
+    def __init__(self, instrumentation: p.Instrumentation) -> None:
         self._instrumentation = instrumentation
+        self._instrumentation.freeze()
 
-    def __call__(self, args: Tuple[Any, ...], kwargs: Dict[str, Any], data: ArrayLike) -> Candidate:
-        return Candidate(args, kwargs, data)
-
-    def from_call(self, *args: Any, **kwargs: Any) -> Candidate:
+    def fr2om_call(self, *args: Any, **kwargs: Any) -> p.Instrumentation:
```

Comment:
typo I assume?

Reply:
Not exactly, that was to trigger mypy and tell me if this was still used somewhere -> it's not :D I'll revert indeed though and add deprecation warnings...

Reply:
Oh smart!